LondonBritishLibrary/orient/BLorient11719.xml

```diff
@@ -0,0 +1,337 @@
+                                        <dimensions type="outer" unit="mm">
+                                            <height>153-173</height>
+                                            <width>115-120</width>
+                                        </dimensions>
+                                        <note>Dimensions vary with 173mm x 115mm on <locus from="3" to="48"/>, 
+                                            153mm x 115mm on <locus from="49" to="64"/>, 162mm x 120mm on <locus from="65" to="72"/> and
+                                            165mm x 120mm on <locus from="74" to="80"/>. It cannot be excluded, that these represent different 
+                                            codicological units, even though they appear to be inscribed with the same texts and partly with the same layout.
+                                        </note>
+                                    </extent>
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="33"><locus from="3" to="64"/></layout>
+                                    <layout columns="1" writtenLines="24 37"><locus from="65" to="72"/></layout>
+                                    <layout columns="1" writtenLines="27 34"><locus from="74" to="80"/></layout>
+                                </layoutDesc>   
+                            </objectDesc> 
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h2">
+                                    <locus from="3r" to="65v"/>
+                                    <desc>Mediocre handwriting, rather small.</desc>
+                                    <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn">18th century</date>
+                                    <note>Possibly more than one hand.</note>
+                                </handNote>
+                                <handNote script="Ethiopic" xml:id="h3">
+                                    <locus from="66v" to="80v"/>
+                                    <desc>Mediocre handwriting, a little larger and inclined.</desc>
+                                    <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>#
+                                    <note>Possibly more than one hand.</note>
+                                </handNote>
+                            </handDesc> 
+                        </physDesc>    
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1799">18th century</origDate>    
+                            </origin>
+                            <provenance>The owner was <persName ref="PRS14491WaldaMaryam">Walda Māryām</persName>, the scribe or 
+                                one of the scribes was <persName ref="PRS14492WaldaGabreel"><roleName type="title">ʾabbā</roleName> 
```

Comment:
ʾAbbā ?

Reply:
I thought, that titles are regularly small, but probably this rule applies only for titles in PRS records.